src/dotnet/APIView/APIViewWeb/Managers/CodeFileManager.cs

```diff
@@ -79,19 +79,22 @@ public async Task<CodeFile> GetCodeFileAsync(string repoName,
                     foreach (var entry in archive.Entries)
                     {
                         var fileName = Path.GetFileName(entry.Name);
-                        if (fileName == originalFileName)
+                        if (!string.IsNullOrEmpty(fileName))
```

Comment:
Given it's a rare case, would it sometimes be unexpected? That is to say, is it worth logging when this is empty?

Reply:
That would be if all the folders are empty and there is no content in the artifact at all. Otherwise, it should just skip empty folders without informing the user.